ops/l2os-entrypoint.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/sh
```

Comment:
I think it's nice to reduce code, and don't like the curl at startup, but why exactly does this PR remove these sh files?

Reply:
The best practice for Docker containers is to treat them like executables. They should usually expose a single `CMD` that people can then pass env vars or CLI flags.

We violated this best practice in order to make sure that dependent services were online before starting the main executable in the container. The devnet-up script does this for us now, so we can remove these entrypoint scripts.